packages/account/src/Sections/Security/AccountDeactivated/account-deactivated.jsx

```diff
@@ -21,7 +22,7 @@ const AccountDeactivated = ({ logout }) => {
         if (timer > 0) {
             setTimer(timer - 1);
         } else {
-            window.location.href = getDerivComLink();
+            window.location.href = getStaticUrl('', { is_deriv_crypto });
```

Comment:
```suggestion
            window.location.href = getStaticUrl('/', { is_deriv_crypto });
```
can we have trailing slashes, since we are forcing trailing slashes in our SEO hreflang backlinks as well